chat-plugins/faq.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const ROOMFAQ_FILE = path.resolve(__dirname, '../config/chat-plugins/faqs.json');
+const ALLOWED_HTML = ['a', 'font', 'i', 'u', 'b'];
+
+let roomFaqs = {};
+try {
+	roomFaqs = require(ROOMFAQ_FILE);
+} catch (e) {} // file doesn't exist or contains invalid JSON
+if (!roomFaqs || typeof roomFaqs !== 'object') roomFaqs = {};
+
+function saveRoomFaqs() {
+	fs.writeFileSync(ROOMFAQ_FILE, JSON.stringify(roomFaqs));
+}
+
+exports.commands = {
+	addfaq: function (target, room, user) {
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('declare', null, room)) return false;
+		if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+		let [topic, ...text] = target.split(',');
+
+		topic = toId(topic);
+		if (!(topic && text.length)) return this.parse('/help roomfaq');
+		text = text.join(',').trim();
+		if (topic.length > 25) return this.errorReply("FAQ topics should not exceed 25 characters.");
+		if (text.length > 500) return this.errorReply("FAQ entries should not exceed 500 characters.");
+
+		let htmltags = text.toLowerCase().match(/<\/?(\w+)\b/g);
+		if (htmltags && htmltags.some(val => !ALLOWED_HTML.includes(toId(val)))) {
```

Comment:
Well they can technically bypass it with /eval but I don't want any other tags in here even by admins.